File: discover_ruby/manipulate_the_data.rb
require 'httpclient'
require 'json'

extheaders = {
  'User-Agent' => 'Holberton_School',
  'Authorization' => 'token 89a061b4ed9ecbfd9801450418d3af551b06c4ae'
}

uri = "https://api.github.com/search/repositories?q=language:ruby&sort=stars&order=desc"

clnt = HTTPClient.new
json_string = clnt.get_content(uri, extheaders)
res = JSON.parse(json_string)['items']

names = res.map {|e| e["full_name"]}
puts names.join("\n")

File: discover_more_python/manipulate_the_data.py
import requests

headers = {
  'User-Agent': 'Holberton_School',
  'Authorization': 'token 89a061b4ed9ecbfd9801450418d3af551b06c4ae'
}  
for item in requests.get('https://api.github.com/search/repositories?q=language:python&sort=stars&order=desc', headers=headers).json()['items']:
	print item['full_name']
